psl-core/src/main/java/org/linqs/psl/application/inference/online/OnlineServer.java

```diff
@@ -242,12 +269,36 @@ private void initializeConnection() {
             }
         }
 
+        private void sendModelInformation() {
+            // Send Client model information for action validation.
+            List<Predicate> predicates = new ArrayList<Predicate>(Predicate.getAll());
+            List<Predicate> modelInformationPredicates = new ArrayList<Predicate>();
+
+            // Remove Functional Predicates.
```

Comment:
Make this comment a bit more accurate.
Something like: "Add non-functional predicates."